api/structure/schema.py

```diff
@@ -0,0 +1,31 @@
+import graphene
+from graphql import GraphQLError
+from graphene_sqlalchemy import SQLAlchemyObjectType
+from sqlalchemy import func
+from helpers.auth.authentication import Auth
+from api.structure.models import Structure as StructureModel
+
+
+class Structure(SQLAlchemyObjectType):
+    class Meta:
+        model = StructureModel
+
+
+class Query(graphene.ObjectType):
```

Comment:
Good work @rajeman .It would be better if you added descriptions for these two queries.

Reply:
@TeamoreA I have done that